components/name.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+
+export default function Name(props) {
+  const [name, setName] = useState("");
+  const { setNameTF } = props;
+
+  useEffect(() => {
+    name === ""
+      ? setNameTF("")
+      : setNameTF(
+          `l_text:v1644177732:Inscryption:HEAVYWEIGHT.ttf_72:${encodeURIComponent(
+            name
+          )},g_north,y_180/c_scale,`
+        );
+  }, [name]);
+
+  return (
+    <section className="mb-14">
+      <p className="text-5xl text-orange-400 font-title text-shadow-orange mb-3">

```

Comment:
you have these exact classes for all the big text: `text-5xl text-orange-400 font-title text-shadow-orange mb-3`

Could be worth extracting to a custom class so that if you add more fields and need to change them in the future, you can change just one class instead of going through each p tag and changing it. Especially if you're extracting to individual components.

Reply:
I'm very, very new to Tailwind so might be wrong here - but my understanding is the standard with TW is to not use custom CSS, and rely completely on TW as far as possible?

Done some reading here: https://www.themes.dev/blog/typographic-defaults-in-tailwind-css

& there's a cool plugin for letting you set typographic defaults. Will switch to this & add a card, but that's unrelated to this PR

Reply:
https://trello.com/c/0TQApNMd/26-provide-typographic-defaults-for-text-instead-of-long-duplicated-class-lists card added